.github/workflows/ci.yml

```diff
@@ -29,6 +29,17 @@ jobs:
       - name: Initialize tests
         run: make initialize -j 4
 
+      - name: Get composer cache directory
+        id: composer-cache
+        run: echo "::set-output name=dir::$(composer config cache-files-dir)"
+
+      - name: Cache Composer
+        uses: actions/cache@v2
+        with:
+          path: ${{ steps.composer-cache.outputs.dir }}
+          key: composer-${{ github.sha }}R
```

Comment:
Why R?

Reply:
`shit - r` to reload changes in my `tig` (my git editor) ^^ 
The focus were in the wrong window...